packages/@biomejs/biome/README.fr.md

```diff
@@ -108,16 +108,35 @@ Nous utilisons [Polar.sh](https://polar.sh/biomejs) pour voter pour des fonction
 
 ## Sponsors
 
+### Sponsors Or
```

Comment:
I think it should be:

```suggestion
### Sponsors d'Or
```

@Conaclos ?

Reply:
As a french native speaker, "Sponsor Or" is fine (both works)